src/main/java/Deadlines.java

```diff
@@ -0,0 +1,19 @@
+public class Deadlines extends Task{
+    private String by;
```

Comment:
Perhaps can put as protected instead of private, so that the member of the same package can access its values instead of using a getter